engine/table/src/main/java/io/deephaven/engine/table/impl/select/MatchFilter.java

```diff
@@ -103,60 +101,33 @@ public MatchFilter(
     public MatchFilter(
             @NotNull final String columnName,
             @NotNull final Object... values) {
-        this(null, null, null, CaseSensitivity.IgnoreCase, MatchType.Regular, columnName, null, values);
+        this(null, CaseSensitivity.IgnoreCase, MatchType.Regular, columnName, null, values);
     }
 
     public MatchFilter(
             @NotNull final CaseSensitivity sensitivity,
             @NotNull final MatchType matchType,
```

Comment:
This addition seems suspicious. You pointed out that we seem to have removed one constructor and added another.